mcs/class/System/Test/System.Diagnostics/PerformanceCounterTest.cs

```diff
@@ -0,0 +1,176 @@
+// from MSDN
```

Comment:
This test is really not so great and maybe remove it in deference to the "s" version in another PR, which covers more code and has actual asserts.

Reply:
Or combine them. Neither is a superset of the other.